### PR TITLE
[WORKFLOWS-449] Add bbsplit to Sarek 3.2.2

### PR DESCRIPTION
**Pull request by danlu1**
Problem:

Need to add [bbmap_bbsplit](https://nf-co.re/modules/bbmap_bbsplit) to Sarek v3.2.2

Solution:
1. Pulled 3.2.2 tag and created a branch v3.2.2-sage
2. Added bbsplit module following this [instruction](https://nf-co.re/docs/contributing/tutorials/adding_modules_to_pipelines) and what was done in [rnaseq pipeline](https://github.com/nf-core/rnaseq)
3. Cross-check the bbsplit output from the modified sarek pipeline and rnaseq with the SAME input
4. Included instructions on how to use bbsplit in Sarek

Note:
Test profile was modified by adding bbsplit related params and genome reference(bbsplit_fasta_list) for testing. Since there is no bbsplit test dataset available for Sarek, I use bbsplit_fasta_list from rnaseq test profile for now. 

Cross-check:
1. the output results are the same with Same input, fasta, bbsplit_fasta_list
2. Code for cross-check will be included in another branch.
Screenshot 2024-03-19 at 9 20 54 PM
Screenshot 2024-03-19 at 9 20 04 PM
Screenshot 2024-03-19 at 9 19 15 PM
Screenshot 2024-03-19 at 9 12 49 PM
Screenshot 2024-03-19 at 9 12 24 PM

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -76,7 +79,7 @@
                     "fa_icon": "fas fa-clock",
                     "description": "Estimate interval size.",
                     "help_text": "Intervals are parts of the chopped up genome used to speed up preprocessing and variant calling. See `--intervals` for more info. \n\nChanging this parameter, changes the number of intervals that are grouped and processed together. Bed files from target sequencing can contain thousands or small intervals. Spinning up a new process for each can be quite resource intensive. Instead it can be desired to process small intervals together on larger nodes. \nIn order to make use of this parameter, no runtime estimate can be present in the bed file (column 5). ",
-                    "default": 200000
+                    "default": 200000.0
```
Comment (thomasyu888) [inline nextflow_schema.json]: Nit: What is the rationale behind this .0?

**Review by thomasyu888 — APPROVED** [review]
🔥 Great work here Dan!  I'm happy for us to merge this as we wait for the example dataset (late April) timeline

Although - not sure why the linting is failing. Can we look into that?

**Comment by danlu1** [comment]
> 🔥 Great work here Dan! I'm happy for us to merge this as we wait for the example dataset (late April) timeline
> 
> Although - not sure why the linting is failing. Can we look into that?

**Comment by danlu1** [comment]
The linting errors are mainly due to the refactor of pipelines template which removes lib/ (see [here](https://nf-co.re/blog/2024/tools-2_13).) Some other errors are to files (such as assets/multiqc_config.yml) that I didn’t change but they match what is in Sarek v3.2.2.

**Pull request merged**